### PR TITLE
Adding Methods to Process EK60 Data

### PR DESCRIPTION
**Pull request by cwingard**
Multiple edits, including creating an environment.yml file for creating a test environment, to the code in order to add EK60 processing. Lots to go through, but the main changes focus on making the changes work across the different sites and types.

### CODE REVIEW

File: zpls_echograms/zpls_echogram.py [hunk 1]
```diff
@@ -2,131 +2,189 @@
...
         'deployed_depth': 450
     },
     'CP03ISSM': {
-        'long_name': 'Inshore Surface Mooring',
+        'long_name': 'Coastal Pioneer, Inshore Surface Mooring',
         'tilt_correction': 15,
-        'colorbar_range': [-150, 0],
+        'colorbar_range': [-90, -50],
         'vertical_range': [0, 90],
         'deployed_depth': 90
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py]: Deployed depth and max vertical range should be 95

File: zpls_echograms/zpls_echogram.py [hunk 2]
```diff
@@ -2,131 +2,189 @@
...
     },
     'CP01CNSM': {
-        'long_name': 'Central Surface Mooring',
+        'long_name': 'Coastal Pioneer, Central Surface Mooring',
         'tilt_correction': 15,
-        'colorbar_range': [-150, 0],
+        'colorbar_range': [-90, -50],
         'vertical_range': [0, 130],
         'deployed_depth': 130
```
Comment (snwhite) [inline zpls_echograms/zpls_echogram.py]: Deployed depth and max vertical range should be 135

**Review by snwhite — CHANGES_REQUESTED** [review]
There are a few default values for CP moorings that we should correct now.  Also wondering why the S/N and Deployment number were pulled out?  Were these only used in the titles and there was not enough room?

**Comment by cwingard** [comment]
@snwhite The EK60 raw files do not include the serial number, nor does RCA use a deployment number. In order to keep the file naming convention the same for all platforms, I pulled those out. I don't see those as critical to the filename, but they can easily be added back in. It just means we are going to have to create two different file naming conventions. Your call, I have no strong opinions one way or the other.

File: zpls_echograms/zpls_echogram.py (hunk 2 above)
Comment (cwingard) [reply zpls_echograms/zpls_echogram.py]: PR updated.

**Review by snwhite — APPROVED** [review]

**Review by kuo-james — APPROVED** [review]

**Review by asmith75218 — APPROVED** [review]

**Pull request merged**